activiti-core/activiti-bpmn-model/src/main/java/org/activiti/bpmn/model/MultiInstanceLoopCharacteristics.java

```diff
@@ -22,7 +22,7 @@ public class MultiInstanceLoopCharacteristics extends BaseElement {
   private String completionCondition;
   private String elementVariable;
   private String elementIndexVariable;
-  private boolean sequential;
+  private Boolean sequential;
```

Comment:
Why change to it a `Boolean`? sequential should be `false` by default. Now there needs to lots of null checks.